### DISCUSSION TITLE
Copied VM's share same identifier in moonlight, how to generate a new one or copy to another machine?

### DISCUSSION THREAD

**Discussion by Hawxxer** [LizardByte/community#529] [Sunshine]
### Select Topic Area

Question

### Body

Hi,
I use sunshine to stream my server vm's under proxmox to my main pc, and I love it, thank you very much! When I copy a vm, which has sunshine installed already, the new vm will use the same identifier as the second vm. This is a behaivor I sometimes use, othertimes it would be awesome to have a new identifier. I guess somewhere is a config file with a unique identifier, could you tell me where? 
Again thank you!

**Comment by ReenigneArcher** [comment]
You need to remove your config files and restart Sunshine

[a comment by an automated account is not shown]
